maat-court-data-api/src/main/java/gov/uk/courtdata/util/StringUtils.java

```diff
@@ -0,0 +1,9 @@
+package gov.uk.courtdata.util;
+
+public final class StringUtils {
```

Comment:
I think we can reuse this from Apache commons lang, the behaviour looks to be the same including if input string is null then the output is also null

`org.apache.commons.lang3.StringUtils.truncate("input string", maxLength)`

Reply:
Sweet